### DISCUSSION TITLE
lspconfig中 rootDir怎么配置

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [nshen/learn-neovim-lua#96] [Q&A]
![图片](https://user-images.githubusercontent.com/43649186/184603952-0a211442-0c43-4034-a098-5f73c42dbe1e.png)
这个see source file啥意思

**Comment by nshen** [comment]
是lsp server 安装目录吧，用lspinstaller安装的好像自动设置 了，不需要这个参数

**Reply by HUAHUAI23** [reply]
这个是配置语言服务器的选项配置，好像是用指定模式来查找项目根目录的

**Reply by HUAHUAI23** [reply]
不知道这个选项咋配置

**Reply by nshen** [reply]
应该是language server的目录，跟项目没关吧
